### DISCUSSION TITLE
How to stop receiving "System Warnings" email?

### DISCUSSION THREAD

**Discussion by afatac** [YetiForceCompany/YetiForceCRM#17334] [How to]
After upgrading to 6.5, I keep receiving this "System Warnings" email every few days. How to disable it?

**Comment by rskrzypczak** [comment]
It's simple, just set 'Ignore' to the appropriate notification type you don't want to receive.
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/742ed218-5894-415c-907e-0553e2f3fe1e)

.. although it would be best to simply correct these comments ;)
